ros2/src/airsim_ros_pkgs/src/airsim_ros_wrapper.cpp

```diff
@@ -1404,6 +1429,48 @@ std::shared_ptr<sensor_msgs::msg::CompressedImage> AirsimROSWrapper::get_jpeg_ms
     return jpeg_message;
 }
 
+std::shared_ptr<image_metadata::Metadata> AirsimROSWrapper::get_jpeg_msg_metadata()

```

Comment:
is this the metadata being written into the images we will get for digital twin? in that case should we not write the 360 normalized data that is global? I are gimbal angles global here? as in, for example gimbal_attitude->yaw is from north?

Reply:
Yes, this function is used by `get_jpeg_msg_from_img_msg` to get the metadata, which adds it to the image before publishing.

Vehicle and gimbal attitudes are global.